src/style-sheets/layout/_hamburger-toolbar.scss

```diff
@@ -42,6 +42,7 @@
       width: 100%;
 
       & div.search-box-wrapper {
+        display: none; // v-2
```

Comment:
Can you add TODO's wherever we're hiding things for v2? Would help us find things faster and keeps things scoped

Reply:
Sure thing!